fleet/lib/rook-ceph-conf/charts/ruka/templates/cephobjectstoreuser-loki.yaml

```diff
@@ -0,0 +1,33 @@
+---
+apiVersion: ceph.rook.io/v1
+kind: CephObjectStoreUser
+metadata:
+  name: loki
+  namespace: rook-ceph
+spec:
+  store: lfa
+  clusterNamespace: rook-ceph
+---
+apiVersion: external-secrets.io/v1alpha1
+kind: PushSecret
+metadata:
+  name: rook-ceph-object-user-lfa-loki
+  namespace: rook-ceph
+spec:
+  secretStoreRefs:
+    - kind: ClusterSecretStore
+      name: onepassword-oods
+  selector:
+    secret:
+      name: rook-ceph-object-user-lfa-loki
+  data:
+    - match:
+        secretKey: AccessKey
+        remoteRef:
+          remoteKey: loki
```

Comment:
I would rename this to `loki-dev` to refer to the site this is going to.

Reply:
Ok, you're right.